src/anm/media/audio.js

```diff
@@ -321,19 +321,21 @@ Audio.prototype.toggleMute = function() {
     }
 };
 /** @private @method connect */
-Audio.prototype.connect = function(element) {
+Audio.prototype.connect = function(player) {
     var me = this;
-    element.on(C.X_START, function() {
+    player.on(C.S_PLAY, function() {
```

Comment:
Please consider this code: https://github.com/Animatron/player/blob/2964e9bc1833137b07204d23ac0d292c6219ab0b/src/anm/media/audio.js#L323 `C.X_START`/`C.X_STOP` are Element band events, `C.S_START`/`C.S_STOP` are Player (or Animation band) events, all four are required.